docker/px4-dev/Dockerfile_ros2-crystal

```diff
@@ -54,12 +54,17 @@ RUN curl https://bootstrap.pypa.io/get-pip.py | python3 \
 		pytest-runner \
 		pytest-rerunfailures
 
-# Build Fast-RTPS release 1.7.0 from source and replace fastrtpsgen
```

Comment:
Removing FastRTPS?

Reply:
Yes this is already handled in px4-dev-base.